gapid_tests/resource_creation_tests/CreateTrimResetDestroyCommandPool_test/main.cpp

```diff
@@ -24,7 +24,8 @@ int main_entry(const entry::EntryData* data) {
   data->logger()->LogInfo("Application Startup");
   vulkan::LibraryWrapper wrapper(data->allocator(), data->logger());
   vulkan::VkInstance instance(
-      vulkan::CreateEmptyInstance(data->allocator(), &wrapper));
+      // vkTrimCommandPool should only be called with VK version 1.1 and greater
```

Comment:
This means this test now cannot run on any vulkan 1.0 device. We should split out the vkTrimCommandPool test to have its own instance (and possibly be in its own file)

Reply:
If we're shifting the vkTrimCommandPool commands to a separate file, what would be a proper directory to put it? I don't know whether `gapid_tests/resource_creation_tests` would be appropriate for a test centered around vkTrimCommandPool.

Reply:
Probably the command_buffer_control file.

Reply:
I don't think I can find any file named command_buffer_control under the gapid_tests. Is it somewhere else?

Reply:
Sorry, command_buffer_tests.